plugins/woocommerce/src/Admin/API/Reports/Products/DataStore.php

```diff
@@ -379,7 +380,11 @@ public function get_noncached_data( $query_args ) {
 
 			$this->subquery->clear_sql_clause( 'select' );
 			$this->subquery->add_sql_clause( 'select', $selections );
-			$this->subquery->add_sql_clause( 'order_by', $this->get_sql_clause( 'order_by' ) );
+			if ( in_array( $query_args['orderby'], array( 'items_sold', 'net_revenue', 'orders_count' ), true ) ) {
```

Comment:
```suggestion
			if ( in_array( $query_args['orderby'], array( 'items_sold', 'net_revenue', 'orders_count', 'variations' ), true ) ) {
```
Should we add `variations`? Users have the option to sort by "variations," and I can see CSV file has duplicate rows in the exported CSV file when sorting by "variations."

<img width="679" height="435" alt="Screenshot 2025-08-12 at 15 17 43" src="https://github.com/user-attachments/assets/dd62742f-64fb-4c9e-8602-454819294d8c" />

Reply:
Good catch, will do!

Reply:
Updated: https://github.com/woocommerce/woocommerce/pull/60258/commits/ed179373f08647068142f1d038b667d8007c2999